website/README.md

```diff
@@ -110,6 +110,9 @@ title: This Doc Needs To Be Edited
 My new content here..
 ```
 
+Note: Ensure the file name and the id value do not include non-url safe
+characters i.e. '\*'.
```

Comment:
Updated the website README to make sure future contributors don't use non-url safe characters.